pkg/sql/parser/help_test.go

```diff
@@ -53,9 +53,13 @@ func TestContextualHelp(t *testing.T) {
 		{`ALTER TENANT 1 ??`, `ALTER TENANT`},
 		{`ALTER TENANT 1 SET ??`, `ALTER TENANT`},
 		{`ALTER TENANT 1 RESET ??`, `ALTER TENANT`},
-		{`ALTER TENANT ALL ??`, `ALTER TENANT`},
-		{`ALTER TENANT ALL SET ??`, `ALTER TENANT`},
-		{`ALTER TENANT ALL RESET ??`, `ALTER TENANT`},
+
+		{`ALTER TENANT ALL ??`, `ALTER TENANT CLUSTER SETTING`},
+		{`ALTER TENANT ALL SET ??`, `ALTER TENANT CLUSTER SETTING`},
+		{`ALTER TENANT ALL RESET ??`, `ALTER TENANT CLUSTER SETTING`},
+
+		{`ALTER TENANT 'foo' RESUME REPLICATION ??`, `ALTER TENANT REPLICATION`},
+		{`ALTER TENANT 'foo' PAUSE REPLICATION ??`, `ALTER TENANT REPLICATION`},
```

Comment:
I believe the way to read this test is the more tokens you type on the left the more specific help text you get, atleast thats my understanding 😅 . So an `ALTER TENANT ALL ??` is currently only supported for altering a tenant cluster setting and so that's the help text we see.